src/g_vumeter.c

```diff
@@ -412,8 +412,10 @@ static void vu_label(t_vu *x, t_symbol *s)
 static void vu_label_pos(t_vu *x, t_symbol *s, int ac, t_atom *av)
 {iemgui_label_pos((void *)x, &x->x_gui, s, ac, av);}
 
-static void vu_label_font(t_vu *x, t_symbol *s, int ac, t_atom *av)
-{iemgui_label_font((void *)x, &x->x_gui, s, ac, av);}
+static void vu_label_font(t_vu *x, t_symbol *s, int ac, t_atom *av){
+    iemgui_label_font((void *)x, &x->x_gui, s, ac, av);
+    (*x->x_gui.x_draw)(x, x->x_gui.x_glist, IEM_GUI_DRAW_MODE_CONFIG);
+}
 
```

Comment:
i wonder whether we shouldn't call `draw_config` directly from `iemgui_label_font`.

pros
- it would allow all iemguis to do more sophisticated adjustments when the font changes.
- we already do something very similar when the size changes

this would create a performance penalty when changing the label-font for all iemguis, but i think font-changes are done seldom enough to not be a real problem.

Reply:
on re-thinking, your proposed solution is probably better than my suggestions.